apps/modernization-ui/src/apps/page-builder/components/TableMenu/TableMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import { Input } from 'components/FormInputs/Input';
+import { Button } from '@trussworks/react-uswds';
+import { Icon } from '@trussworks/react-uswds';
+import { useNavigate } from 'react-router-dom';
+
+type Props = {
+    tableType: string;
+};
+
+export const TableMenu = ({ tableType }: Props) => {
+    const navigate = useNavigate();
+    const onClick = () => {
+        navigate(`/page-builder/add-new-${tableType}`);
+    };
```

Comment:
should the ${tableType} be a path parameter instead of appending to the url?

Reply:
we could do it like `/page-builder/add/${page}` and make the routes that way--that what you're thinking?

Reply:
yup